queries/zig/highlights.scm

```diff
@@ -27,13 +27,19 @@
 
 (build_in_call_expr
   function: ((identifier) @include
-             (#eq? @include "@import"))
+             (#any-of? @include "@import" "@cImport"))
+)
+(struct_construction
+  (type_identifier) @constructor
 )
-
 ;; other identifiers
 (type_identifier) @type
+(custom_number_type) @type.builtin
 (primitive_type) @type.builtin
 (field_identifier) @field
+(enum_identifier) @field
```

Comment:
I think we have an `enum` type that should be used here.

Reply:
The enum_identifier is the green box in the image below, i think it should be either **constant** or **field**

![enum](https://user-images.githubusercontent.com/34153891/127725597-d12b4df2-764c-48ab-9288-daac226fa756.jpg)

Reply:
I'd rather have that a constant, as this is not exactly a field... Sorry to b annoying :wink:

Reply:
I tried to find what is [highlight](https://github.com/nvim-treesitter/nvim-treesitter/blob/master/lua/nvim-treesitter/highlight.lua) for enum type is, is it a `@type.enum`.

Reply:
Then that's not the one we are searching for, indeed.

Reply:
> I think we have an `enum` type that should be used here.

I just want to know what is the `enum` type you mentioned above. For the sake of improve me knowledge.

Reply:
It was just a mistake from me, I thought `enum` existed, which is not true :wink:

Reply:
Could you make that a `constant` though ? As this is technically a constant ?

Reply:
I have a day off tomorrow. I will make it

Reply:
It's a constant now, i have more highlight, should i make a separate pull request or add more commit to this pull request, or combine it in one commit

Reply:
As you wish, we can make that go further already, and see for the rest afterwards ?

Reply:
![Untitled - Frame 0](https://user-images.githubusercontent.com/34153891/128146723-d9eb57c0-437a-4351-bfd5-755632d710b1.jpg)

Reply:
Great !